src/app/saga.ts

```diff
@@ -364,10 +378,76 @@ export function* handleSetAppState(action: SetAppState) {
   }
 }
 
+export function* runCentralPhoneVerificationMigration() {
+  const privateDataEncryptionKey = yield select(dataEncryptionKeySelector)
+  if (!privateDataEncryptionKey) {
+    Logger.warn(
+      `${TAG}@runCentralPhoneVerificationMigration`,
+      'No data encryption key was found in the store. This should never happen.'
+    )
+    return
+  }
+
+  const shouldRunVerificationMigration = yield select(shouldRunVerificationMigrationSelector)
+  if (shouldRunVerificationMigration) {
```

Comment:
nit: return early for less nesting.
```suggestion
  if (!shouldRunVerificationMigration) {
    return
  }
```